.napari/DESCRIPTION.md

```diff
@@ -0,0 +1,96 @@
+# Description
+
+This GUI plugin allows you to quickly find the affine matrix mapping
+one image to another using manual correspondence points annotation.
+
+More simply, this plugin allows you to select corresponding points
+on an image, and a second image you wish to transform. It computes 
+the requisite transformation matrix using Affine Transform, Euclidean Transform, 
+or Similarity Transform, and performs this transformation on the
+moving image, aligning it to the reference image.
+
+![Affinder aligning two images of a parasitic worm](https://i.imgur.com/FIhnqa0.gif)
```

Comment:
Please remove this gif.

😜😜😜😜😜

Reply:
![ApprehensiveTornIndigobunting-small](https://user-images.githubusercontent.com/17995243/120086524-f34eab00-c122-11eb-9db4-7418e979569c.gif)